app-backend/common/src/test/scala/com/implicits/Generators.scala

```diff
@@ -951,7 +951,15 @@ object Generators extends ArbitraryInstances {
       projectId <- uuidGen
       projectLayerId <- uuidGen
       status <- taskStatusGen
-    } yield { Task.TaskPropertiesCreate(projectId, projectLayerId, status) }
+      annotationProjectId <- uuidGen
```

Comment:
Similar comment to above, but replace "routes" with "prior steps in the testing transaction"